src/crawlee/_types.py

```diff
@@ -399,6 +399,10 @@ async def push_data(
         **kwargs: Unpack[PushDataKwargs],
     ) -> None:
         """Track a call to the `push_data` context helper."""
+        from crawlee.storages._dataset import Dataset
```

Comment:
What is the justification for not having import on top of the file?

Reply:
It would introduce an import cycle.